install/kubernetes/helm/istio/values.yaml

```diff
@@ -92,6 +92,11 @@ sidecar-injector:
   # be allowed by the sidecar
   includeIPRanges: {}
 
+  # Force admission registration webhook when using `helm template`
```

Comment:
If siedcar-injector is enabled - I think we should add the registration as well (no need to have 2 booleans to control this). It won't work without it.

Reply:
I originally made them separate to support the old initializer method as well as this but given that MutatingWebhooks will be the real mechanism going forward I'm happy to change this.